src/main/java/frc/robot/Constants.java

```diff
@@ -145,13 +145,17 @@ public static final class ElevatorConstants {
     public static final int ENCODER_CHANNEL_B = 1;
     public static final int LIMIT_SWITCH_CHANNEL = 2;
     public static final int ELEVATOR_MOTOR_ID = 31;
-    public static final double kP = 1.3;
     public static final double kMaxVelocity = 0.75;
     public static final double kMaxAcceleration = 0.5;
-    public static final double kDistancePerPulse = 0;
-    public static final double kMaxExtension = 10;
+    public static final double kDistancePerPulse = 5.5 / 2048;
```

Comment:
Noting that the `5.5` value comes from 22 teeth per sprocket multiplied by 1/4" per chain link.  Larger sprockets or a different chain size would mean a different constant here.